### PR TITLE
docs: adds java 8 and java 11 compatibility note for ksqlDB

### PR DESCRIPTION
**Pull request by cprasad1**
### Description 
Adds java 8 and java 11 compatibility note for ksqlDB to the docs.

### Testing done 
_Describe the testing strategy. Unit and integration tests are expected for any behavior changes._

### Reviewer checklist
- [ ] Ensure docs are updated if necessary. (eg. if a user visible feature is being added or changed).
- [ ] Ensure relevant issues are linked (description should include text like "Fixes #<issue number>")

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ ksqlDB requires only:
 We recommend using [Confluent
 Platform](https://www.confluent.io/product/confluent-platform/) or
 [Confluent Cloud](https://www.confluent.io/confluent-cloud/) to run
-{{ site.ak }}.
+{{ site.ak }}. ksqlDB only supports Java 8 and 11.
```
Comment (JimGalasyn) [inline docs/faq.md:31]:
```suggestion
{{ site.ak }}. ksqlDB supports Java 8 and 11.
```

File: docs/operate-and-deploy/installation/installing.md [hunk 2]
```diff
@@ -378,6 +378,8 @@ versions.
 | {{ site.aktm }} version | 0.11.0 and later         |
 | {{ site.cp }} version   | 5.5.0 and later          |
 
+ksqlDB only supports Java 8 and Java 11.
```
Comment (JimGalasyn) [inline docs/operate-and-deploy/installation/installing.md:381]:
```suggestion
ksqlDB supports Java 8 and Java 11.
```

**Review by JimGalasyn — APPROVED** [review]
LGTM, with a couple of tweaks.

File: docs/faq.md [hunk 3]
```diff
@@ -28,7 +28,7 @@ ksqlDB requires only:
 We recommend using [Confluent
 Platform](https://www.confluent.io/product/confluent-platform/) or
 [Confluent Cloud](https://www.confluent.io/confluent-cloud/) to run
-{{ site.ak }}.
+{{ site.ak }}. ksqlDB supports Java 8 and 11.
```
Comment (agavra) [inline docs/faq.md:31]: Out of interest, where did you get this information? We only compile on java 8, but I thought any jdk runtime above 8 should be able to run the compiled JAR (I have not verified).

**Pull request merged**